server/py/services/api/api/endpoints/model_monitoring.py

```diff
@@ -107,7 +107,15 @@ async def _common_parameters(
     )
 
 
-@router.post("/enable-model-monitoring")
+# TODO: remove /projects/{project}/model-monitoring/enable-model-monitoring in 1.8.0
```

Comment:
```suggestion
# TODO: remove /projects/{project}/model-monitoring/enable-model-monitoring in 1.10.0
```

Reply:
As we talked, lets remove the deprecated paths in the end of 1.8.0.
I will open a task for it.